calendarsubscription/calendarsubscription.go

```diff
@@ -46,3 +50,45 @@ func (cs CalendarSubscription) Normalize() (*CalendarSubscription, error) {
 
 	return &cs, nil
 }
+
+func (cs CalendarSubscription) renderICalFromShifts(shifts []oncall.Shift, reminderMinutes []int) ([]byte, error) {
```

Comment:
`reminderMinutes` is already available in `cs.Config.ReminderMinutes`, no need for the param